src/PdfReader/index.tsx

```diff
@@ -459,17 +480,28 @@ export default function usePdfReader(args: ReaderArguments): ReaderReturn {
         const index = manifest?.readingOrder?.findIndex((link) => {
           return link.href === href;
         });
-        if (!index) {
+        if (index === undefined || index === -1) {
```

Comment:
huh, this is tricky.  This will return "false" if index is `null` so it's not quite there.  I _think_ 
`if (Number(index)) < 1) // or 0, the fact that pages that are 1-indexed is messing me up`  might give you what you want.

Reply:
`!(Number(0) > 0)` returns true, so this doesn't work

Reply:
That's an indexing problem, right?  If 0 is valid, then it would be `!(Number(index) >= 0)`

Reply:
Since PDFs start at page=1 it messes with me every time I look at this repo